### DISCUSSION TITLE
[SMTP] How to provide nocodb with correct setting

### DISCUSSION THREAD

**Discussion by stepanof** [nocodb/nocodb#1689] [Q&A] [answered]
Hello team. I'm trying to set up SMTP app to be able to send invition email.
When I use Secure = true/false I get error in both cases.
![image](https://user-images.githubusercontent.com/68236577/163663541-ce2ce250-1bc3-47fd-bf4f-cd1834a8adde.png)
![image](https://user-images.githubusercontent.com/68236577/163663553-cfb5042e-ce31-4846-8792-3b4d744ef1bf.png)
I would be grateful for a hint where to look for the cause of errors and how to set up SMTP correctly.
What value should be set for Secure paramert?

**Comment by wingkwong** [comment]
The possible value for secure is either `true` or `false`. I think the setup in your first screenshot should be fine. The error states that you also need to enter username and password.

**Comment by stepanof** [comment, marked as answer]
@wingkwong  Thanks for your answer!

Now I suppose I have found bag in new 0.90 version.

How to reproduce:
1) Set up SMTP app in App store. Test connection.
![image](https://user-images.githubusercontent.com/68236577/163691956-063264af-8f7a-4bb4-8956-e05c033dbf50.png)

2) Try to invite new member. 
See notification that email setting wasn't set up yet.
![image](https://user-images.githubusercontent.com/68236577/163691838-e2e6ede7-8045-4597-ad13-11d016c7da92.png)

3) Have a look on the SMTP setting - the value of checkbox 'Ignore TLS' wasn't saved.
![image](https://user-images.githubusercontent.com/68236577/163691889-a09d71b2-8246-4d7e-8102-0dd344bcaa58.png)

Project info:
Node: **v12.22.12**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **true**
RootDB: **pg**
PackageVersion: **0.90.0**

**Comment by stepanof** [comment]
I have just tested the same steps in the same environment but with 0.84.16 version.
Everything works perfect.
